src/javascript/flow/language.ts

```diff
@@ -74,13 +74,6 @@ export default class FlowGenerator {
       keyInheritsNullability: true
     });
 
-    if (description) {
```

Comment:
This PR [is failing due to a linting error](https://travis-ci.org/apollographql/apollo-codegen/jobs/319459408#L501):
> ```
> src/javascript/flow/language.ts(61,19): error TS6133: 'description' is declared but its value is never read.
> ```

I suspect changing line 61 above will fix it:
```diff
- const { name, description } = inputObjectType;
+ const { name } = inputObjectType;
```